tests/macro_utils_int_tests.c

```diff
@@ -23,26 +23,44 @@ int main(void)
     test1 someUnusedVariable;
     (void)someUnusedVariable;
 
-    if (ENUM_TO_STRING_2(TEST3_ENUM, 0) != NULL)
+    const char* TEST2_value_0 = ENUM_TO_STRING_2(TEST3_ENUM, 0);
+    if (TEST2_value_0 == NULL)
     {
-        printf("TEST2_ENUM should not know how to compute value 0\n");
+        (void)printf("TEST2_ENUM should not return NULL\n");
         result++;
     }
+    else
+    {
+        if (strcmp("NULL", TEST2_value_0) != 0)
+        {
+            (void)printf("ENUM_TO_STRING_2(TEST2_ENUM, 0) should return NULL\n");
```

Comment:
>NULL [](start = 72, length = 4)

wpuld be nice to have this enclosed in \"NULL\", just to be able to tell NULL from "NULL"

Reply:
Done.

---
In reply to: [226868551](https://github.com/Azure/azure-macro-utils-c/pull/4#discussion_r226868551) [](ancestors = 226868551)